pkgs/by-name/re/readest/package.nix

```diff
@@ -55,7 +55,6 @@ rustPlatform.buildRustPackage (finalAttrs: {
     substituteInPlace src-tauri/Cargo.toml \
       --replace-fail '"devtools"' '"devtools", "rustls-tls"'
     substituteInPlace src-tauri/tauri.conf.json \
-      --replace-fail '"createUpdaterArtifacts": true' '"createUpdaterArtifacts": false' \
```

Comment:
The hook from this PR won't work for readest, because the [tauri.conf.json](https://github.com/readest/readest/blob/38e24da96882b9bbab021d9eb2d5913902436d80/apps/readest-app/src-tauri/tauri.conf.json) file is not stored in the `cargoRoot` directory.